confirm-buypack-dialog.user.js

```diff
@@ -0,0 +1,39 @@
+// ==UserScript==
+// @name        FUT Confirm Buy Pack Dialog
+// @version     0.1.0
+// @description Automatically confirm the Buy Pack dialog
+// @license     MIT
+// @author      Maciej 'kielon' Adamczyk
+// @match       https://www.easports.com/fifa/ultimate-team/web-app/*
+// @match       https://www.easports.com/*/fifa/ultimate-team/web-app/*
+// @namespace   https://github.com/Mardaneus86
+// @updateURL   https://raw.githubusercontent.com/Mardaneus86/futwebapp-tampermonkey/master/confirm-buypack-dialog.user.js
+// @downloadURL https://raw.githubusercontent.com/Mardaneus86/futwebapp-tampermonkey/master/confirm-buypack-dialog.user.js
+// @supportURL  https://github.com/Mardaneus86/futwebapp-tampermonkey/issues
+// ==/UserScript==
+// ==OpenUserJS==
+// @author kielon
+// ==/OpenUserJS==
+(function () {
+    'use strict';
+  
+    var targetNodes         = $(document);
+    var MutationObserver    = window.MutationObserver || window.WebKitMutationObserver;
+    var myObserver          = new MutationObserver (mutationHandler);
+    var obsConfig           = { childList: true, characterData: true, attributes: false, subtree: true };
+  
+    targetNodes.each ( function () {
+        myObserver.observe (this, obsConfig);
+    } );
+  
+    function mutationHandler (mutationRecords) {
+      mutationRecords.forEach ( function (mutation) {
+        if ($(mutation.addedNodes).hasClass('Dialog')) {
+          if (gPopupClickShield._activePopup._title === "store.confirmation.questionTitle") {
```

Comment:
Title value doesn't look unique.
Are you sure this test bypass only buy pack dialog?